tests/testComposite.py

```diff
@@ -50,41 +65,47 @@ def tearDown(self):
         if os.path.exists(self.output):
             shutil.rmtree(self.output)
 
-    def test(self):
+    def testGet(self):
         """Verify that a composite can be loaded and that its components are the same as when the type1
         components are loaded individually (verifies correct lookup in this case).
+
         Also verify that when the individual components are put and when the composite is put (which
         disassembles into individual components) that the objects that are written are the same.
         """
+        butler = dafPersist.Butler(
+            inputs=dafPersist.RepositoryArgs(root=self.input, mapper='lsst.obs.test.testMapper.TestMapper'),
+        )
+        rawAndFlat = butler.get('rawAndFlat', dataId={'visit': 1, 'filter': 'g'})
+        raw = butler.get('raw', dataId={'visit': 1, 'filter': 'g'})
+        flat = butler.get('flat', dataId={'filter': 'g'})
+        self.assertEqual(pickle.dumps(raw), pickle.dumps(rawAndFlat.raw))
```

Comment:
Also, these are just testing that the individually-read thing (raw, flat) is the same as the group-read thing (rawAndFlat). Is there some way to test them more thoroughly, or is this enough? Or, wait, that's the whole point of this test suite, no?